basicswap/basicswap.py

```diff
@@ -11618,6 +11620,17 @@ def getWalletInfo(self, coin):
                 rv["mweb_pending"] = (
                     walletinfo["mweb_unconfirmed"] + walletinfo["mweb_immature"]
                 )
+            elif coin == Coins.FIRO:
+                try:
+                    rv["spark_address"] = self.getCachedStealthAddressForCoin(Coins.FIRO)
+                except Exception as e:
+                    self.log.warning(
+                        f"getCachedStealthAddressForCoin for {ci.coin_name()} failed with: {e}."
+                    )
+                # Spark balances are in atomic units, format them
+                rv["spark_balance"] = ci.format_amount(walletinfo["spark_balance"], conv_int=True)
+                spark_pending_int = walletinfo["spark_unconfirmed"] + walletinfo["spark_immature"]
+                rv["spark_pending"] = ci.format_amount(spark_pending_int, conv_int=True)
```

Comment:
```suggestion
                rv["spark_balance"] = ci.format_amount(walletinfo["spark_balance"])
                spark_pending_int = walletinfo["spark_unconfirmed"] + walletinfo["spark_immature"]
                rv["spark_pending"] = ci.format_amount(spark_pending_int)
```

the conv_int causes display issues here